metadata.json

```diff
@@ -29,7 +29,6 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "18.04",
         "20.04",
```

Comment:
Looks like 20.04 has also reached EOL

Reply:
I checked before I opened the PR and 18.04 has a month to go: https://ubuntu.com/about/release-cycle